libraries/Microsoft.Bot.Builder.AI.QnA/QnAMakerOptions.cs

```diff
@@ -1,11 +1,14 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
+
 namespace Microsoft.Bot.Builder.AI.QnA
 {
     /// <summary>
     /// Defines options for the QnA Maker knowledge base.
     /// </summary>
+    [Serializable]
```

Comment:
I don't think you want this - this is the old-school BINARY serialization attribute. Not needed (that I can remember) with the Json approach you're using.

Reply:
Ah, you're right! Works w/o it :)